deploy/kubernetes/base/controller/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
+- controller.yaml
+
+commonLabels:
+  k8s-app: gcp-compute-persistent-disk-csi-driver
```

Comment:
In the deleted kustomization.yaml, the label used is
`app: gcp-compute-persistent-disk-csi-driver`
Do you want to keep using it?

Reply:
Working on the new version of the PR now. The label key has been changed to `k8s-app` in the base